cmd/ascii2der/scanner.go

```diff
@@ -284,6 +284,51 @@ again:
 		if s.pos.Offset+1 < len(s.text) && s.text[s.pos.Offset+1] == '"' {
 			return s.parseUTF32String()
 		}
+	case 'b':
+		if s.pos.Offset+1 < len(s.text) && s.text[s.pos.Offset+1] == '`' {
+			s.advance() // Skip the b.
+			s.advance() // Skip the `.
+			bitStr, ok := s.consumeUpTo('`')
+			if !ok {
+				return token{}, &parseError{s.pos, errors.New("unmatched `")}
+			}
+
+			// Leading byte is the number of "extra" bits at the end.
```

Comment:
Nit: s/Leading/The leading/ for a complete sentence. I guess since the logic below describes it as padding bits, we can probably s/"extra" bits/padding bits/.